helm/distributed-idol/values.yaml

```diff
@@ -11,7 +11,7 @@
 # This section contains parameters that are most likely to be overridden for
 # the basic choices and constraints around how this chart will be used.
 
-# setupMirrored is a boolean parameter - when `true` this will configure the 
+# -- When `true` this will configure the 
```

Comment:
Are we deliberately dropping compliance with https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml (_"The documentation string should begin with the name of the property that it describes"_) here?

(Not convinced by the motivation given, but at one point we had a ticket to update the values.yaml to conform with the guide...)

Reply:
Yes deliberate. I think it's more useful to comply with comment format required for https://github.com/norwoodj/helm-docs?tab=readme-ov-file#valuesyaml-metadata  and generate more useful documentation that way.